src/main.rs

```diff
@@ -31,8 +33,38 @@ use vmm::vmm_config::instance_info::{InstanceInfo, InstanceState};
 
 const DEFAULT_API_SOCK_PATH: &str = "/tmp/firecracker.socket";
 const DEFAULT_INSTANCE_ID: &str = "anonymous-instance";
+const KERNEL_VERSION: f64 = 4.14;
+
+fn is_kvm_writable() -> Result<bool, Error> {
+    let kvm_rw = Command::new("sh")
+        .arg("-c")
+        .arg("[ -r /dev/kvm ] && [ -w /dev/kvm ] && echo OK|| echo FAIL")
+        .output()
+        .expect("failed to execute process");
+    Ok(str::from_utf8(&kvm_rw.stdout[0..2]).unwrap() == "OK")
+}
+fn kernel_version() -> Result<f64, Error> {
+    let kernel_version = Command::new("sh")
```

Comment:
Same here. You can use the [`uname()` syscall](https://linux.die.net/man/2/uname) instead of running an external binary.

Reply:
If you go for `uname`, whitelist it in `default_syscalls.rs` otherwise seccomp will block it out when running jailed.